### ISSUE TITLE
IPC mucus 

### ISSUE THREAD

**Issue by staticcharged** [HeavensGate/Eternal#333]
IPCs will get the text "Mucus runs down the back of your throat"

It's an IPC, they don't have bodily fluids. Or throats.

**Comment by Mewchild** [comment]
This is part of a larger issues, in which viruses are able to target /mob/carbon/human/machine, a class they should be completely excluded from. Alternatively, computer viruses able to be caught by them and /silicon entities, with an alternative set of effects. Biological viruses shouldn't be able to be caught by metal men.

**Comment by Davnik** [comment]
I agree with this issue being made URGENT. It does, however, need to be referenced to #280.

**Comment by Davnik** [comment]
Note that #280 will fix this better,  and prevent dozens of other theoretical issues that revolve around IPC's getting viruses: coughing, sweating, etc.
